lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -710,6 +710,20 @@ bool firrtl::areTypesEquivalent(FIRRTLType destFType, FIRRTLType srcFType,
     return true;
   }
 
+  // Enum types can be connected if they have the same size, element names, and
+  // element types.
```

Comment:
This says they're equivalent not the same? Can you connect less-inferred to more-inferred?

From the comment, I'd expect this to be source == dest enum type.

Reply:
Yeah youre right, that is confusing. To be clear, you can connect less-inferred to more inferred. I based it on the other comments in this function which commit the same sin.  I'll fix it in a follow up if that is alright.

Reply:
Sounds good to me!  Thanks.